app/Services/Api/ApiEntityService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Services\Api;
+
+use App\Models\MiscModel;
+use App\Models\Campaign;
+
+class ApiEntityService
```

Comment:
This class name doesn't really describe what it does. Better would be `App\Services\Api\BulkEntityCreatorService` or something similar